SU2_CFD/src/solver_structure.cpp

```diff
@@ -226,7 +226,8 @@ void CSolver::SetResidual_RMS(CGeometry *geometry, CConfig *config) {
       
     }
     
-    SetRes_RMS(iVar, max(EPS*EPS, sqrt(rbuf_residual[iVar]/Global_nPointDomain)));
+    SetRes_RMS(iVar, max(EPS*EPS, sqrt(rbuf_residual[iVar]/ geometry->GetnPoint() )));
```

Comment:
I´m not sure about this change. @economon What do you think?

Reply:
Could you be a little bit more specific about why the result is wrong in serial? I have run a quick test with the QuickStart test and I think I always have the same number of points 5233 (for serial of parallel simulations). Thanks!